### PR TITLE
III-3926 events with placetypes

### PR DESCRIPTION
**Pull request by JonasVHG**
### Changed

- allow a fallback for events with a `placeType` instead of an `eventType`

---
Ticket: https://jira.uitdatabank.be/browse/III-3926

### CODE REVIEW

File: src/Event/ReadModel/JSONLD/EventLDProjector.php [hunk 1]
```diff
@@ -316,7 +320,15 @@ protected function applyEventCopied(
         foreach ($eventJsonLD->terms as $term) {
             if ($term->domain === 'eventtype') {
                 $typeId = new StringLiteral($term->id);
-                $eventType = $this->eventTypeResolver->byId($typeId);
+                // This is a workaround to allow copies of events that
+                // have a placeType instead of an eventType.
+                // These events could also be cleaned up in the future
+                // @see https://jira.uitdatabank.be/browse/III-3926
```
Comment (bertramakers) [inline src/Event/ReadModel/JSONLD/EventLDProjector.php:326]: Very clear! 👌

File: tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php [hunk 2]
```diff
@@ -194,6 +194,47 @@ public function it_handles_new_events_without_theme(): void
         $this->assertEquals($jsonLD, $body);
     }
 
+    /**
+     * @test
+     */
+    public function it_handles_new_events_with_an_incorrect_place_type(): void
+    {
+        $eventId = '1';
+        $eventCreated = new EventCreated(
```
Comment (bertramakers) [inline tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php:203]: Shouldn't this test `EventCopied` instead? 🤔

Comment (JonasVHG) [reply tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php:203]: Indeed, I've refactored the test

File: src/Event/ReadModel/JSONLD/EventLDProjector.php (hunk 1 above)
Comment (JonasVHG) [reply src/Event/ReadModel/JSONLD/EventLDProjector.php:326]: Thanks!

**Review by bertramakers — APPROVED** [review]
Thanks for the fix @JonasVHG ! 🥇

**Pull request merged**